partners/project-05.md

```diff
@@ -0,0 +1,36 @@
+---
+altLangPage: "https://conception.canada.ca/partenaires/project-05.html"
+date: 2023-12-08
+dateModified: 2023-12-08
+description: "Partner project: Help Health Canada prepare for implementing search functionality in a new Drupal-based RSAMS publishing infrastructure, to be launched in the 2020 to 2021 fiscal year. Date: Fall 2019"
+title: "Recalls and safety alerts search"
+---
+<p>Help Health Canada prepare for the implementation of a transformed Recalls and Safety Alerts Management System (RSAMS) with a new search interface so that Canadians can find and understand both recalls and safety alerts.</p>
+<h2>Date</h2>
+<p>Fall 2019</p>
+<h2>Partner departments</h2>
+<ul>
+  <li>Health Canada</li>
+  <li>Transport Canada</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>the federal government is the largest public buyer of goods and services in Canada, with purchases of approximately $23 billion annually</li>
```

Comment:
This is incorrect. It should be: "There are thousands of recalls and safety alerts published every year. When Canadians can find and understand this information, they are safer."

Reply:
Fixed.